pccommon/setup.py

```diff
@@ -14,6 +14,7 @@
     "cachetools==5.0.0",
     "types-cachetools==4.2.9",
     "pyhumps==3.5.3",
+    "redis==4.2.0-rc1",
```

Comment:
4.2.0-rc1 is the first version with async capabilities. Should move to released version when available.